app/views/snps/index.html.erb

```diff
@@ -10,16 +10,16 @@
       <p class="text-center snps__text-download">Includes annotation for all SNPs from all sources</p>
     </div>
   </div>
-  <div class="table-responsive">
```

Comment:
Don't know whether the comment made it: but it should stay responsive as it's about the design for mobiles/tablets